.idea/.idea.RavenM.dir/.idea/discord.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please uncommit the JetBrains files and add .idea to the .gitignore instead.

Reply:
I may have accidentaly commited the .idea folder while tryong to squash the other commits